authors/apps/articles/tests/test_favorite_article.py

```diff
@@ -0,0 +1,171 @@
+import json
+from django.urls import reverse
+from django.test import TestCase
+from rest_framework.test import APIClient
+from authors.apps.authentication.models import User
+from ..models import Article
+from rest_framework import status
+
+
+
+class FavoriteArticle(TestCase):
+    """
+    Tests for Favoring Articles
+    """
+
+    def setUp(self):
+        self.email = 'johndoe@gmail.com'
+        self.username = 'johndoe'
+        self.password = 'Johndoe3@'
+
+
+        self.email2 = 'fieldmarshal@gmail.com'
+        self.username2 = 'fieldmarshal'
+        self.password2 = 'FieldMarshal@3'
+        # create a user
+        self.user = User.objects.create_user(
+            self.username, self.email, self.password)
+
+        # create the second user
+        self.user2 = User.objects.create_user(
+            self.username2, self.email2, self.password2)
+
+        # verify a user's account and save
+        self.user.is_verified = True
+        self.user.save()
+        self.token = self.user.token
+
+        # verify and save the second user
+        self.user2.is_verified = True
```

Comment:
I do not know of this field in the user model.

Reply:
@bisonlou Thanks for the feedback , this field is supposed to be 'is_email_verified'. let me fix this.